db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] The previous reviewer asked for this. Please use the seeds file to create 5 messages that will become their own randomly fetched messages by the front-end app. For example:
```ruby
Message.create(greetings: 'Message No.1')
Message.create(greetings: 'Message No.2')
Message.create(greetings: 'Message No.3')
Message.create(greetings: 'Message No.4')
Message.create(greetings: 'Message No.5')
```